modules/geo-layers/src/great-circle-layer/great-circle-vertex.glsl.js

```diff
@@ -69,9 +74,14 @@ vec2 interpolate (vec2 source, vec2 target, float angularDist, float t) {
 
 	float a = sin((1.0 - t) * angularDist) / sin(angularDist);
 	float b = sin(t * angularDist) / sin(angularDist);
-	float x = a * cos(source.y) * cos(source.x) + b * cos(target.y) * cos(target.x);
-	float y = a * cos(source.y) * sin(source.x) + b * cos(target.y) * sin(target.x);
-	float z = a * sin(source.y) + b * sin(target.y);
+  vec2 sin_source = sin(source);
+  vec2 cos_source = cos(source);
+  vec2 sin_target = sin(target);
+  vec2 cos_target = cos(target);
+
+  float x = a * cos_source.y * cos_source.x + b * cos_target.y * cos_target.x;
+  float y = a * cos_source.y * sin_source.x + b * cos_target.y * sin_target.x;
+  float z = a * sin_source.y + b * sin_target.y;
```

Comment:
Line 77 - 84 indent not correct

Reply:
It's just github formatting tabs weirdly